userguide/conf.py

```diff
@@ -87,6 +87,10 @@
 .. |ctrlr-term-1-2|       replace:: TrueNAS controller 1/2
 .. |ctrlr-term-active|    replace:: active TrueNAS controller
 .. |ctrlr-term-standby|   replace:: standby TrueNAS controller
+.. |ctrlrs-term-both|     replace:: active and standby TrueNAS controllers
+.. |active|               replace:: active
+.. |standby|              replace:: standby
+.. |actv-stndby|          replace:: active/standby
```

Comment:
Abbreviating this one when the others are not abbreviated is inconsistent and will probably lead to errors.